src/sentry/api/endpoints/organization_member_invite/details.py

```diff
@@ -140,7 +145,90 @@ def put(
 
         return Response(serialize(invited_member, request.user), status=200)
 
+    def _remove_invite_from_db(
```

Comment:
this is almost the same as `reject_invite_request` on the `OrganizationMemberInvite` model. could they be collapsed into a single function?